src/miniflask/miniflask.py

```diff
@@ -1368,6 +1372,26 @@ def like(self, varname, alt, scope="."):
             scope, _ = self._get_relative_module_id(scope)
         return like(varname, alt, scope=scope, scope_name=scope_name)
 
+    def as_is_callable(self, variable):
+        r"""
+        Wrap variables for register_-calls to ensure they are not parsed during initialization.
+
+        # Note {.alert}
+        This function is typically useful in case a module saves a callable object upon initialization inside a helper variable.
+
+        Args:
+        - `varname`: The variable to use.
+
+        Examples:
```

Comment:
As this is a rather advanced topic I have a minor suggestion.
Can we add the result after the variable is registered (with `mf.as_is_callable`) to the example, and maybe additionally what would happen without this to compare both strategies?

Reply:
Good point, I've expanded on that in the newest (force-pushed) commit.